### PR TITLE
New Files Added : JavaScript assignment 7 & 9

### PR DESCRIPTION
**Pull request by Pranjali1999**

### CODE REVIEW

File: Pranjali/JavaScript/Assi7/assi7_2.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    </head>
+    <body>
+        <div style="width: 40%; position: absolute; left: 500px; top: 200px">
+            <div style="text-align: center; background-color: rgba(236, 236, 146, 0.911); height: 50%; border-radius: 25px 25px 25px 25px">
+                <script language="javascript">
+                    var n = Number(prompt("Enter the total elements in an Array"));
+                    var arr = {};
+                    var i;
+                    document.write('<div style="position: relative; top: 20px"><h1>Print Marks<h1></div><br><hr size="5" color=black><br><br>');
+
```
Comment (akash19972306) [inline Pranjali/JavaScript/Assi7/assi7_2.html]: remove extra line

File: Pranjali/JavaScript/Assi7/assi7_5.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    function printArray(arr, n) {
+                        for (let i = 0; i < n; i++) {
+                            document.write(arr[i] + "");
+                            document.write("<br>");
+                        }
+                    }
+                    InsertionSort(arr, n);
+                    printArray(arr, n);
+                    //document.write("<br> <h2>After Insertion Sort " + arr + "</h2>");
+                    console.log(arr, n);
```
Comment (akash19972306) [inline Pranjali/JavaScript/Assi7/assi7_5.html:51]: remove comment line

File: Pranjali/JavaScript/Assi9/assi9_1.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    </head>
+    <body>
+        <p style="color: blue"></p>
+        <script language="javascript">
+            function Palnum() {
+                var rem,
+                    temp,
+                    final = 0;
+                var number = Number(document.getElementById("N").value);
+
```
Comment (akash19972306) [inline Pranjali/JavaScript/Assi9/assi9_1.html]: remove extra line

File: Pranjali/JavaScript/Assi9/assi9_1.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                temp = number;
+                while (number > 0) {
+                    rem = number % 10;
+                    number = parseInt(number / 10);
+                    final = final * 10 + rem;
+                }
+                if (final == temp) {
+                    //window.alert("number is Palindrome");
+                    //document.write("number is Palindrome");
```
Comment (akash19972306) [inline Pranjali/JavaScript/Assi9/assi9_1.html:31]: remove comment line

**Review by akash19972306 — CHANGES_REQUESTED** [review]

**Review by akash19972306 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**